.github/ISSUE_TEMPLATE/release.md

```diff
@@ -4,32 +4,62 @@ about: Prepare for a release
 labels: maintenance
 ---
 
-- [ ] merge all outstanding PRs
-- [ ] ensure the versions have been bumped (check with `doit`)
-- [ ] ensure the CHANGELOG is up-to-date
-  - [ ] move the new release to the top of the stack
-- [ ] validate on binder
-- [ ] validate on ReadTheDocs
+- [ ] on `dev`
+  - [ ] merge all outstanding PRs
+    - [ ] _blocking #PR here_
+  - [ ] ensure the versions have been bumped
+    - [ ] run `doit preflight:release`
+      - [ ] `pyproject.toml`
```

Comment:
I know this is mostly for us but I was confused for a second why the `pyproject.toml` and `package.json` were under the `preflight:release`, would it be better to explicitly add another check that says, inspect files with hard-coded versions, and then under that have the `pyproject.toml` and the `package.json` bullets?

Reply:
sure, will get rid of it.